clear-containers-image/update_image.sh

```diff
@@ -6,37 +6,55 @@
 # Default image to build is the one specified in file versions.txt
 # located at the root of the repository.
 set -x
+if [ $# -ne 1 ]; then
+    cat << EOT
+Usage:
+$0 <clear-linux-version>
+
+You need to provide the version to update the image
+EOT
+    exit
+fi
+script_dir=$(dirname "$0")
```

Comment:
scripts_dir points to the same directory as cc_image_dir. Can we use just one variable?

Reply:
is not, `script_dir` could be `./`and `cc_image_dir` is `/home/foo/packaging/clear-containers-image`

Reply:
I see, script_dir is useful when calling the script from any other directory, which is uncommon.

Reply:
+1